src/EnergyPlus/DaylightingManager.cc

```diff
@@ -5414,10 +5414,15 @@ void GetInputDayliteRefPt(EnergyPlusData &state, bool &ErrorsFound)
         pt.Name = state.dataIPShortCut->cAlphaArgs(1);
         pt.ZoneNum = UtilityRoutines::FindItemInList(state.dataIPShortCut->cAlphaArgs(2), state.dataHeatBal->Zone);
         if (pt.ZoneNum == 0) {
-            ShowSevereError(state,
-                            cCurrentModuleObject + "=\"" + state.dataIPShortCut->cAlphaArgs(1) + "\", invalid " +
-                                state.dataIPShortCut->cAlphaFieldNames(2) + "=\"" + state.dataIPShortCut->cAlphaArgs(2) + "\".");
-            ErrorsFound = true;
+            int spaceNum = UtilityRoutines::FindItemInList(state.dataIPShortCut->cAlphaArgs(2), state.dataHeatBal->space);
```

Comment:
Fix Daylighting:ReferencePoint to accept a space name. At this point, it's only used to map to a zone number, which gets used for any coordinate transformations. Spaces don't have their own coordinates (yet?).